src/components/OrgNode.vue

```diff
@@ -44,11 +43,6 @@ export default {
       }
     }
   },
-  computed: {
```

Comment:
I suspect something happened to replace this? Can't see what it is, but if there is a replacement, I'm good with that! 🎉